Dockerfile.kafkaInit

```diff
@@ -0,0 +1,59 @@
+FROM openjdk:11-jdk-slim as builder
+
+RUN mkdir -p /code/java-sdk
+WORKDIR /code/java-sdk
+
+COPY commons /code/commons
+COPY specifications /code/specifications
+COPY java-sdk/gradle /code/java-sdk/gradle
+COPY java-sdk/build.gradle java-sdk/settings.gradle  java-sdk/gradlew /code/java-sdk/
+
+ENV GRADLE_OPTS -Dorg.gradle.daemon=false
+
+RUN ./gradlew tasks
+COPY java-sdk/radar-schemas-commons/build.gradle /code/java-sdk/radar-schemas-commons/
+COPY java-sdk/radar-schemas-commons/src /code/java-sdk/radar-schemas-commons/src
+RUN ./gradlew :radar-schemas-commons:jar
+COPY java-sdk/radar-schemas-tools/build.gradle /code/java-sdk/radar-schemas-tools/
+COPY java-sdk/radar-schemas-tools/src /code/java-sdk/radar-schemas-tools/src
+RUN ./gradlew distTar && cd radar-schemas-tools/build/distributions && tar xzf radar-schemas-tools*.tar.gz
+
+
+FROM openjdk:8-alpine
+
+ENV KAFKA_SCHEMA_REGISTRY=http://schema-registry-1:8081
+ENV KAFKA_NUM_PARTITIONS=3
+ENV KAFKA_NUM_REPLICATION=2
+ENV KAFKA_NUM_BROKERS=3
+ENV KAFKA_ZOOKEEPER_CONNECT=zookeeper-1:2181
+
+RUN apk add --no-cache \
+  bash \
+  curl \
+  rsync \
+  tar
+
+RUN mkdir -p /schema/merged /schema/java/src /schema/java/classes /usr/share/java
+
+WORKDIR /schema
+
+ENV JQ_VERSION=1.5
+RUN curl -L#o /usr/bin/jq https://github.com/stedolan/jq/releases/download/jq-${JQ_VERSION}/jq-linux64 \
+    && chmod +x /usr/bin/jq
+RUN curl -#o /usr/share/java/avro-tools.jar \
+ "$(curl -s http://www.apache.org/dyn/closer.cgi/avro/\?as_json \
+  | jq --raw-output ".preferred")avro/avro-1.8.2/java/avro-tools-1.8.2.jar"
```

Comment:
Avro 1.8.2 is not not available anymore at that path. It has been archived. Please take a look at https://github.com/RADAR-base/RADAR-Docker/pull/192

Reply:
Sorry. My bad